vizro-core/src/vizro/models/_components/form/radio_items.py

```diff
@@ -54,15 +54,15 @@ def build(self):
         return html.Div(
             [
                 dbc.Label(self.title, html_for=self.id) if self.title else None,
-                dcc.RadioItems(
+                dbc.RadioItems(
                     id=self.id,
                     options=full_options,
                     value=self.value if self.value is not None else default_value,
                     persistence=True,
                     persistence_type="session",
-                    className="radio-items-list",
+                    className="form-check",
                 ),
             ],
-            className="input-container",
+            className="form-group",
```

Comment:
@pruthvip15 - not sure if these are the right classNames to use. It depends on the SASS code you wrote and where you've customised it.

Reply:
Bootstrap uses form-check, hence the classnames, i am not sure about the form group yet

<img width="832" alt="Screenshot 2024-04-11 at 17 08 56" src="https://github.com/mckinsey/vizro/assets/121864270/7c436291-143a-45a6-a027-9f90e4986c03">

Reply:
@pruthvip15 - do we have to provide the form-group then?